cmd/edgectl/aes_install.go

```diff
@@ -119,65 +119,107 @@ func aesInstall(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func getManifest(url string) (string, error) {
+	res, err := http.Get(url)
+	if err != nil {
+		return "", err
+	}
+	bodyBytes, err := ioutil.ReadAll(res.Body)
+	res.Body.Close()
+	if err != nil {
+		return "", err
+	}
+	if res.StatusCode != 200 {
+		return "", errors.Errorf("Bad status: %d", res.StatusCode)
+	}
+	return string(bodyBytes), nil
+}
+
 func (i *Installer) Perform(kcontext string) error {
+	// Start
 	i.Report("install")
 
+	// Download AES manifests
+	crdManifests, err := getManifest("https://www.getambassador.io/yaml/aes-crds.yaml")
+	if err != nil {
+		i.Report("fail_no_internet", ScoutMeta{"err", err.Error()})
+		return errors.Wrap(err, "download AES CRD manifests")
+	}
+	aesManifests, err := getManifest("https://www.getambassador.io/yaml/aes.yaml")
+	if err != nil {
+		i.Report("fail_no_internet", ScoutMeta{"err", err.Error()})
+		return errors.Wrap(err, "download AES manifests")
+	}
+
+	// Figure out what version of AES is being installed
+	// TODO: Parse the manifests and build objects
+	// TODO: Extract version info from the Deployment object
```

Comment:
Comment seems outdated

Reply:
The TODO comments are still valid, sadly.